src/app.html

```diff
@@ -15,25 +15,5 @@
 	</head>
 	<body class="min-h-full text-gray-50 bg-gray-950 font-sans" data-sveltekit-preload-data="hover">
 		<div style="display: contents">%sveltekit.body%</div>
-		<script>
-			(function(a,b,c){var d=a.history,e=document,f=navigator||{},g=localStorage,
-			h=encodeURIComponent,i=d.pushState,k=function(){return Math.random().toString(36)},
-			l=function(){return g.cid||(g.cid=k()),g.cid},m=function(r){var s=[];for(var t in r)
-			r.hasOwnProperty(t)&&void 0!==r[t]&&s.push(h(t)+"="+h(r[t]));return s.join("&")},
-			n=function(r,s,t,u,v,w,x){var z="https://www.google-analytics.com/collect",
-			A=m({v:"1",ds:"web",aip:c.anonymizeIp?1:void 0,tid:b,cid:l(),t:r||"pageview",
-			sd:c.colorDepth&&screen.colorDepth?screen.colorDepth+"-bits":void 0,dr:e.referrer||
-			void 0,dt:e.title,dl:e.location.origin+e.location.pathname+e.location.search,ul:c.language?
-			(f.language||"").toLowerCase():void 0,de:c.characterSet?e.characterSet:void 0,
-			sr:c.screenSize?(a.screen||{}).width+"x"+(a.screen||{}).height:void 0,vp:c.screenSize&&
-			a.visualViewport?(a.visualViewport||{}).width+"x"+(a.visualViewport||{}).height:void 0,
-			ec:s||void 0,ea:t||void 0,el:u||void 0,ev:v||void 0,exd:w||void 0,exf:"undefined"!=typeof x&&
-			!1==!!x?0:void 0});if(f.sendBeacon)f.sendBeacon(z,A);else{var y=new XMLHttpRequest;
-			y.open("POST",z,!0),y.send(A)}};d.pushState=function(r){return"function"==typeof d.onpushstate&&
-			d.onpushstate({state:r}),setTimeout(n,c.delay||10),i.apply(d,arguments)},n(),
-			a.ma={trackEvent:function o(r,s,t,u){return n("event",r,s,t,u)},
-			trackException:function q(r,s){return n("exception",null,null,null,null,r,s)}}})
-			(window,"G-2GKZ82L9E5",{anonymizeIp:true,colorDepth:true,characterSet:true,screenSize:true,language:true});
-		</script>
```

Comment:
Could you please move this change to a separate PR or issue? I think the scope of the current PR #3 should be adding the new features of engineering mode and bluetooth control. The discussion around feature additions will be quite different to the discussion around removing analytics, and therefore they're best to live in separate places.

For the record: I appreciate and agree with this removal because

- Google Analytics is heavy-handed and totally overkill for this site.
- GA is misconfigured on my end and this tracking code is therefore currently useless for actually getting an idea of page visit count (which is the only thing I want to be aware of as the maintainer and host of this site)